lib/dpl/version.rb

```diff
@@ -1,3 +1,3 @@
 module DPL
-  VERSION = '1.8.32'
+  VERSION = '1.8.33'
```

Comment:
Please remove this. We manage the releases, and we cannot merge this PR with this bit.